lib/shelley/bench/Restore.hs

```diff
@@ -195,43 +196,45 @@ cardanoRestoreBench tr c socketFile = do
                 np
                 vData
                 "rnd.timelog"
-                walletRnd)
+                (walletRnd mkRndState))
 
-        , bench ("restore " <> network <> " 1% ownership")
-            (bench_restoration @_ @IcarusKey
+        , bench ("restore " <> network <> " 1% rnd")
+            (bench_restoration @_ @ByronKey
                 networkProxy
                 tr
                 socketFile
                 np
                 vData
-                "1-percent.timelog"
-                (initAnyState "Benchmark 1% Wallet" 0.01))
+                "1-percent-rnd.timelog"
+                (walletRnd $ mkRndAnyState @1))
 
-        , bench ("restore " <> network <> " 2% ownership")
```

Comment:
Removing because it was too slow, I presume?

Maybe it's worth adding `0.1%`, or `0.5%`, to still have more data points than 0% and 1%.

(Useful, I imagine, if you'd ever want to do some curve fitting on the data)

Reply:
> Removing because it was too slow, I presume?

Correct.

> Maybe it's worth adding 0.1%, or 0.5%, to still have more data points than 0% and 1%.

Good idea :+1: